examples/Console/TestPrometheusExporter.cs

```diff
@@ -16,80 +16,63 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Diagnostics.Metrics;
 using System.Threading;
 using System.Threading.Tasks;
 using OpenTelemetry;
 using OpenTelemetry.Metrics;
 using OpenTelemetry.Trace;
 
-namespace Examples.Console
-{
-    internal class TestPrometheusExporter
-    {
-        private static readonly Meter MyMeter = new Meter("TestMeter");
-        private static readonly Counter<double> Counter = MyMeter.CreateCounter<double>("myCounter", description: "A counter for demonstration purpose.");
-        private static readonly Histogram<long> MyHistogram = MyMeter.CreateHistogram<long>("myHistogram");
-        private static readonly ThreadLocal<Random> ThreadLocalRandom = new ThreadLocal<Random>(() => new Random());
-
-        internal static object Run(int port, int totalDurationInMins)
-        {
-            /*
-            Following is sample prometheus.yml config. Adjust port,interval as needed.
+namespace Examples.Console;
 
-            scrape_configs:
-              # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
-              - job_name: 'OpenTelemetryTest'
+internal class TestPrometheusExporter
+{
+    private static readonly Meter MyMeter = new Meter("MyMeter");
+    private static readonly Counter<double> Counter = MyMeter.CreateCounter<double>("myCounter", description: "A counter for demonstration purpose.");
```

Comment:
please put `unit` as well, so as to demo how it'll appear in prometheus.

Reply:
the thread runaway time has unit so I intend to not put unit here